charts/thoras/templates/reasoning-api/deployment.yaml

```diff
@@ -33,7 +33,15 @@ spec:
       - image: {{ .Values.imageCredentials.registry }}/thoras-forecast:{{ default .Values.thorasVersion .Values.thorasReasoningApi.imageTag }}
         imagePullPolicy: "{{ .Values.imagePullPolicy }}"
         name: thoras-reasoning-api
-        command: ["/usr/local/bin/reasoning_api"]
+        command:
+          - "gunicorn"
+          - "--workers"
+          - "4"
```

Comment:
Hard-coded for now so we can ship the smallest useful thing but we'll want to do something more clever at some point